Examples/downloader/downloader.py

```diff
@@ -0,0 +1,159 @@
+from os import mkdir, makedirs, remove
+from os.path import dirname, isdir, isfile, join
+import pandas as pd
+
+import seerpy
+from tqdm import tqdm
+
+from downloader.utils import write_json, add_to_csv
+
+
+def get_download_log_file():
+    """Retrieves JSON of download errors."""
+    log_file = join(dirname(__file__), 'logger.csv')
+    if not isfile(log_file):
+        pd.DataFrame().to_csv(log_file)
+
+    return log_file
+
+
+class DataDownloader:
+    def __init__(self, client, study_id, path_out):
+        self.client = client
+        self.study_id = study_id
+        self.label_groups = self.get_label_groups()
+        self.channel_groups = self.get_channel_groups()
+
+        self.study_name = self.channel_groups['name'][0]
+        self.folder_out = join(path_out, self.study_name)
+        if not isdir(self.folder_out):
+            mkdir(self.folder_out)
+
+    def get_channel_groups(self):
+        channel_groups = self.client.get_all_study_metadata_dataframe_by_ids(
+            [self.study_id])
+        channel_groups = channel_groups.drop(columns=[
+                'description', 'patient', 'channelGroups.chunkPeriod',
+                'channelGroups.compression', 'channelGroups.timestamped',
+                'segments.timezone', 'channels.channelType.category',
+                'channels.channelType.name'
+            ])
+        return channel_groups
+
+    def get_label_groups(self):
+        return self.client.get_label_groups_for_studies([self.study_id])
+
+    def get_channels(self):
+        channels = {}
```

Comment:
Something like this:
```suggestion
        return self.channel_groups[['channelGroups.name', 'channels.name']].drop_duplicates()
```
I haven't tested this, it may not be exactly right. This will return a dataframe, you can convert it to a list using .to_numpy().tolist()

Reply:
Ah! Okay, I've just added in a change